eng/pipelines/update-azure-linux-pipeline.yml

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# For info about runtime parameters, see https://github.com/microsoft/go-infra/blob/main/docs/pipeline-yml-style.md#runtime-parameters
+parameters:
```

Comment:
The new pipeline should be added to https://github.com/microsoft/go-infra/blob/main/eng/pipelines/README.md so it's easier to navigate to. Sorry I didn't spot this in the first pass.

(Line number not relevant, just wanted a resolve-able thread.)

Reply:
added